### DISCUSSION TITLE
can vkui be used for websites?

### DISCUSSION THREAD

**Discussion by vitalijalbu** [VKCOM/VKUI#3403] [Q&A] [answered]
hi guys, maybe a dumb question, can vkui be used for websites? i mean a really responsive website, here 80% of visitors are from mobile

**Comment by SevereCloud** [comment, marked as answer]
Yes, you can

**Reply by vitalijalbu** [reply]
are there any examples i can see??
I really like the input components, buttons, cards etc... thy are very mobile friendly, that why i'm asking
